modules/@apostrophecms/page/index.js

```diff
@@ -2968,6 +2983,22 @@ database.`);
           });
         });
       },
+      ensureHomeSlugMigration() {
```

Comment:
Actually this happens automatically due to Apostrophe's parked page logic on every startup so it's not necessary to add a migration for it, it's already done just as frequently as migrations are run.